host/4/publish-appservice-stage.yml

```diff
@@ -145,23 +145,6 @@ steps:
     displayName: tag and push node images
     continueOnError: false
 
-  - bash: |
-      set -e
-      docker pull $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-appservice
-      docker pull $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7.2-appservice
-
-      docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-appservice $TARGET_REGISTRY/powershell:$(TargetVersion)-appservice-stage$(StageNumber)
-      docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-appservice $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-appservice-stage$(StageNumber)
-      docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7.2-appservice $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7.2-appservice-stage$(StageNumber)
```

Comment:
This appears to also delete the powershell 7.2 images which I do not think is our aim